job_runner/reporting/event.py

```diff
@@ -87,78 +98,58 @@ def _init_handler(self, msg):
         self._step_id = msg.step_id
         self._event_publisher_thread.start()
 
-    def _job_handler(self, msg):
-        job_path = f"{self._step_path()}/job/{msg.job.index}"
-
+    def _job_handler(self, msg: Message):
+        job_name = msg.job.name()
+        logger.debug(f"Job: {job_name} is being handle")
```

Comment:
I think all outcomes below will result in some debug statement, so this is maybe superfluous.